src/ic-xrc-types/src/lib.rs

```diff
@@ -19,13 +19,14 @@ pub struct Asset {
 }
 
 /// The type the user sends when requesting a rate.
+///
+/// For definitions of "base", "quote", etc, the reader is referred to
+/// https://en.wikipedia.org/wiki/Currency_pair.
 #[derive(CandidType, Clone, Debug, Deserialize)]
 pub struct GetExchangeRateRequest {
-    /// The asset to be used as the resulting asset. For example, using
-    /// ICP/USD, ICP would be the base asset.
+    /// The base asset.
```

Comment:
I think the comment that was here before had some value but now it's a comment that adds no value.

Actually, I think the example was more useful than a link to some long definition.

Reply:
As before, it's up to @THLO to agree or not.

Reply:
Thomas made a suggestion. I'm going to take it.